ethereum/core/src/main/java/org/hyperledger/besu/ethereum/bonsai/BonsaiLayeredWorldState.java

```diff
@@ -283,6 +283,6 @@ public void persist(final BlockHeader blockHeader) {
 
   @Override
   public WorldUpdater updater() {
-    return new BonsaiWorldStateUpdater(this);
+    return new BonsaiWorldStateUpdater(this, (__, ___) -> {}, (__, ___) -> {});
```

Comment:
🤔 Why do we have to pass these as arguments anywhere? If I look at this code in the future, how do I know we did not forget to provide here something? I suggest overloading a constructor and set the NO OP values as defaults on your fields

Reply:
done